packages/manager-overthebox/src/overTheBox.js

```diff
@@ -0,0 +1,60 @@
+import 'ovh-angular-tail-logs';
+import 'angularjs-scroll-glue';
+
+import angular from 'angular';
+
+import constant from './overTheBox.constant';
+import controller from './overTheBox.controller';
+import template from './overTheBox.html';
+
+import ovhManagerOtbConfigure from './configure/overTheBox-configure';
```

Comment:
Maybe we should rename the main js file to `index.js` in each module.

So we could change import to 
```suggestion
import configure from './configure';
```
We are in `overTheBox` context and don't need to prefix all import with it.